<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiMaterial</class>
 <widget class="QWidget" name="UiMaterial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>251</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vLayout_main">
    <item>
     <widget class="QPushButton" name="btn_basicSetting">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>&gt;基础材质设置</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_basicSetting" native="true">
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>50</y>
         <width>121</width>
         <height>111</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="vLayout_pbr">
        <item>
         <widget class="QLineEdit" name="lineEdit_metallic">
          <property name="text">
           <string>1.0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_roughness">
          <property name="text">
           <string>0.1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_ior">
          <property name="text">
           <string>1.450</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>金属度:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>糙度:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>130</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>折射率:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>基础色:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_basiccolor">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>10</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>选择hsv颜色</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>170</y>
         <width>81</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>170</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>自发光:</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
